app/app/src/main/java/de/stustapay/stustapay/ui/account/AccountProperties.kt

```diff
@@ -125,7 +125,7 @@ fun AccountProperties(
                 Divider()
                 SummaryEntry(
                     stringResource(R.string.customer_restriction),
-                    when (account.restriction.value) {
+                    when (account.restriction!!.value) {
```

Comment:
these forced evaluations are timebombs for app-crashes - why do we need them now?